Web Coding Fundamentals/Session 2/functions.js

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1>Functions</h1>
+
+<p>This function will perform a calculation and display the ans:</p>
+
+<p id="multiply"></p>
+
+<script>
```

Comment:
Can we move the script tag and code inside a head tag before the body? If there was more added in next sessions they would all be in the same location?